### DISCUSSION TITLE
Como usar um código de GitHub

### DISCUSSION THREAD

**Discussion by putaprata** [frontendbr/forum#1675] [Discussão]
Olá pessoal, tudo bom? Então, gostaria de usar um código para filtrar itens por preço no site da Wish.com e estou tendo dificuldades. 

Link: https://gist.github.com/dannnylo/de4ae95ff736720965fbbf46a37eef88

Como posso saber se ele está funcionando e como faço pra usa-lo? Tentei abrir o site, cliquei em Inspecionar, console, colei o código e não obtive sucesso

**Comment by portothree** [comment]
Este codigo está criando um Objeto Javascript com algumas funções, você pode acessar com ".", por exemplo: `WishSort.allProductsListed()` ou `WishSort.sort(20)`.
Me parece usar Jquery para os seletores. Se pretende utilizar com um user script em algo como tampermonkey vai precisar importar o Jquery também.

**Comment by putaprata** [comment]
Então, como eu sou novato nesses quesitos, irei dar uma pesquisada para conseguir realizar esses procedimentos. Se puder me auxiliar mais um pouco, terei o prazer de pagar pelo serviço. Agradeço pela resposta :)

**Comment by portothree** [comment]
Tampermonkey é uma extensão para navegadores onde pode inserir estes códigos, no caso do que deixou o link, ele depende de um outro conjunto de códigos chamado Jquery, por isso seria necessário "chamar" este outro conjunto antes.
Mas se quiser tentar antes de instalar o typermonkey, pode ir até à página que deseja ordenar estes itens no wish apertar `CTRL + SHIFT + I` ou botão direito e inspecionar elemento, procurar a opção console no topo e colar este código do gist, dar entender e depois é só escrever `WishSort.sort(20)` na linha de baixo.
